boards/nxp/frdm_mcxa156/frdm_mcxa156_defconfig

```diff
@@ -9,3 +9,4 @@ CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_GPIO=y
+CONFIG_LPADC_DO_OFFSET_CALIBRATION=y
```

Comment:
Does this need to be in the board's base defconfig? Maybe we only enable this in the board's Kconfig.defconfig file if CONFIG_ADC=y?